test/e2e/lib/gutenberg/blocks/blog-posts-block-component.js

```diff
@@ -0,0 +1,11 @@
+/**
+ * Internal dependencies
+ */
+import GutenbergBlockComponent from './gutenberg-block-component';
+
+class BlogPostsBlockComponent extends GutenbergBlockComponent {}
+
+BlogPostsBlockComponent.blockTitle = 'Blog Posts';
+BlogPostsBlockComponent.blockName = 'a8c/blog-posts';
```

Comment:
This seems more idiomatic:

```suggestion
class BlogPostsBlockComponent extends GutenbergBlockComponent {
	static blockTitle = 'Blog Posts';
	static blockName = 'a8c/blog-posts';
}
```

Reply:
Oh, that was the first thing I tried. Unfortunately, I got a syntax error. The tests don't go through a transpiler, so it fallbacks to what node supports, and AFAIK, it still doesn't support `static` class properties.

Reply:
Using node `12.16.2`:

```
 ./node_modules/.bin/mocha specs/wp-calypso-gutenberg-upgrade-spec.js
file:///home/fullofcaffeine/workspace/code/wp-calypso/test/e2e/lib/gutenberg/blocks/youtube-block-component.js:7
        static blockTitle = 'Foo';
                   ^

SyntaxError: Invalid or unexpected token
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1176:10)
```

Reply:
Oh, but it does support static getters:

```javascript
class YoutubeBlockComponent extends GutenbergBlockComponent {
	static get blockTitle() {
		return 'foo ';
	}
}
```

Reply:
I think I'll use static getters for now.

Reply:
I think it's an issue with how this is configured. I believe mocha is using its own parser. In node 12.18.0 it works:

```sh
> class A {}
undefined
> class B { a='a';}
undefined
> A.a
undefined
> B.a
undefined
> new B().a
'a'
> A.a='a'
'a'
> A.a
'a'
> class C { static a = 'a' }
undefined
> C.a
'a'
```

Let's just leave it for this PR. It's not really relevant.

Reply:
Yeah, I tested it with vanilla node and saw it worked. I tried to figure out what as the culprit in mocha but couldn't find and left it aside. I'm using static getters for now.

I'd be interested to know what is the culprit here. I saw the mocha js binary and AFAIK it uses node as well. That's pretty confusing.

Reply:
Fix for this in https://github.com/Automattic/wp-calypso/pull/45268.